tests/fixtures/scenarios/exitCode/stacks/deploy/dev.yaml

```diff
@@ -0,0 +1,11 @@
+vars:
+  stage: dev
```

Comment:
I am not able to remove this line because if i do i get error:
![image](https://github.com/user-attachments/assets/eaf9e500-8a1d-4a0b-9bdb-354d7348ecf4)
This is a bug that we should be fixing. 

Unless there is something else that i had to do

Reply:
I don't understand. Remvoing this line would remove it from `dev`, and the error above is accurate.

Reply:
Do not use the `station` component for testing. Every time we do this, we DoS attack `wttr.in`

Reply:
We are fixing this in another PR, but let's not add new tests usingit.
- #1018

Reply:
oh, @Cerebrovinny is going to refactor that PR.

Reply:
> Do not use the station component for testing. Every time we do this, we DoS attack wttr.in
Nope not DDoS only used the same name for the component. It is empty resource with an echo. so should be fine

Reply:
But we are going to apply this component to test terraform, right?

Reply:
we are just planning... and secondly just using echo.. Also i changed the component name to avoid confusion with `quick-start-simple` and updated the overall atmos workspace to be more minimal. should be better now